### PR TITLE
Settings ui

### PR DESCRIPTION
**Pull request by fitnessGo**
Updated the UI of settings screen
Screenshot_20190711-143415_FitnessGo
image

### CODE REVIEW

File: screens/Settings.js [hunk 1]
```diff
@@ -61,25 +62,59 @@ export default class SettingsScreen extends Component {
     this.setState({ darkTheme: window.darkTheme });
   }
 
+  toggleSwitchIcon() {
+    return (<Icon
+      name='toggle-switch'
+      type='material-community'
+      color={styles.iconDark.color}
+      size={35}
```
Comment (AlexeiGitH) [inline screens/Settings.js]: Let's make all the icons on this screen the same size of 24.

File: screens/Settings.js [hunk 2]
```diff
@@ -97,10 +132,17 @@ const styles = StyleSheet.create({
   },
   textLight: {
     color: '#000000',
-    fontSize: 18
+    fontSize: 18,
```
Comment (AlexeiGitH) [inline screens/Settings.js]: Font size to 16

File: screens/Settings.js [hunk 3]
```diff
@@ -97,10 +132,17 @@ const styles = StyleSheet.create({
...
+
   },
   textDark: {
     color: '#ffffff',
-    fontSize: 18
+    fontSize: 18,
```
Comment (AlexeiGitH) [inline screens/Settings.js]: Font size to 16

**Comment by klymenkoo** [comment]
@fitnessGo @AlexeiGitH Hm, I'm actually not sure about "Switch" icon. 
Tbh, I'd prefer having a native Switch ui element rather than icons. The icon doesn't animate when you click it, which is not a good UX. How difficult would it be to change it back?

**Comment by AlexeiGitH** [comment]
Would be nice if it was animatable as it was with switch control. Not a big of a problem though. Could you try to fix it?

**Pull request closed without merging**